src/Uno.UWP/Devices/Power/BatteryReport.Android.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Windows.Devices.Power
+{
+	public partial class BatteryReport
+	{
+		public int? ChargeRateInMilliwatts;
+		public int? DesignCapacityInMilliwattHours;
+		public int? FullChargeCapacityInMilliwattHours;
+		public int? RemainingCapacityInMilliwattHours;
+		public Windows.System.Power.BatteryStatus Status;
+
+		private Windows.System.Power.BatteryStatus UWPstatusFromAndroStatus(int status)
```

Comment:
```suggestion
		private BatteryStatus UWPstatusFromAndroStatus(int status)
```

Reply:
BatteryStatus is not known in this context, so it has to be prefixed with namespace.